components/button.css

```diff
@@ -1,4 +1,80 @@
+.btn{
+    border-radius: 0.5rem;
+    padding: 0.5rem 1rem;
+    font-weight: 500;
+}
```

Comment:
Looking awesome. Just a suggestion to also add some color accent to button when hovered, it will give good user experience .

Reply:
Thanks for the suggestion @VLeads . I will definitely add this after completing p0.